repos/system_upgrade/el7toel8/actors/multipathconfread/tests/files/already_updated.conf

```diff
@@ -105,7 +105,7 @@ blacklist_exceptions {
 		product "S/390x"
 	}
 }
-		
+
```

Comment:
@ZhukovGreen thinking about that, I guess that the PR should affect only code. it should skip all data files completely. really we should not care about any whitespaces here etc. it's not code, it's just data.

Reply:
@pirat89 it is good to have pre-commit working on all files. There are a lot of useful fixes that might be done for example in yml files or shell scripts. Let's just exclude from linting those files that are outputs of other programs.

Reply:
conf file is not code. from that point, I do not agree we should use linting here. yml files - ok, makes sense, as many people do mistake in yml files. shell scripts are code again, so this doesn't have to be ignored, but again, we should check the filename has expected suffix or shebang. Checking all files, will be from my point of view rather annoying than helpful. can be discussed in team (@oamg/developers)

Reply:
thinking about that, I would skip probably everything inside the "files" directories. these files should not be restricted by us at all.